src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java

```diff
@@ -6,7 +6,28 @@
  */
 package org.mule.extension.ftp;
 
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static org.mule.extension.ftp.AllureConstants.FtpFeature.FTP_EXTENSION;
+
+import java.nio.file.Path;
+import java.util.LinkedList;
+import java.util.Queue;
```

Comment:
Move these imports after the mule imports.
Remember the order of the imports is 
mule
java 
others